generate_website_assets.sh

```diff
@@ -26,6 +26,9 @@ rsync $BUILD/catala.html $1/
 rsync $BUILD/clerk.html $1/
 rsync $BUILD/catala_legifrance.html $1/
 
-rsync $BUILD/french_law/js/french_law.js $1/french_law.js
+# NOTE(@EmileRolley): the french_law.js is accessible through:
+# https://www.npmjs.com/package/@catala-lang/french-law
+# no need to copy it here.
+# rsync $BUILD/french_law/js/french_law.js $1/french_law.js
```

Comment:
You can just delete the line, no need to put the comment. However you should put a README section about how to generate new website assets and how to publish them on NPM.